modules/ztp-precaching-ztp-config.adoc

```diff
@@ -76,7 +75,6 @@ spec:
               macAddress: "AA:BB:CC:11:22:33"
 ----
 <1> Specifies the cluster image set used for deployment, unless you specify a different image set in the `spec.clusters.clusterImageSetNameRef` field.
```

Comment:
<1> we need to remove this statement - "unless you specify a different image set in the `spec.clusters.clusterImageSetNameRef` field" as we don't have a support to provision "spec.clusters.clusterImageSetNameRef` field in OCP 4.12

Reply:
Removed now